src/components/map/DataGroupLine.js

```diff
@@ -41,10 +46,12 @@ const DataGroupLine = ({ line, setPopupVisible, popupVisible }) => {
           style={{
             height: "40px",
             zIndex: popupVisible == line.uuid ? 4 : 3,
+            "--data-group-colour": dataGroupColour,
```

Comment:
This is just a question:
did you test that this works okay when `dataGroupColour` is null, and just defaults to the default `--data-group-colour`?

Reply:
Yep, I also have the Friends Club > Beautiful Data data on my local version and this works fine.